app/services/physical_holdings_markup_builder.rb

```diff
@@ -379,6 +376,9 @@ def self.request_placeholder(adapter, holding_id, location_rules, holding)
     markup
   end
 
+  attr_reader :adapter
```

Comment:
My only comment is the order of this class is now strange.  Aka usually the attr readers and initializer is at the top of the class.

Reply:
Yes, good point, it's worth reorganizing it a bit first. I didn't move them around because I thought it would make the diff harder to read but I'll add it in another commit.